docs/hello_nextflow/03_hello_workflow.md

```diff
@@ -521,22 +521,22 @@ This time the third step was only called once!
 Looking at the output of the `view()` statements, we see the following:
 
 - Three `Before collect:` statements, one for each greeting: at that point the file paths are individual items in the channel.
-- A single `After collect:` statement: the three file paths are now packaged into a single item.
+- A single `After collect:` statement: the three file paths are now packaged into a single element.
 
 Have a look at the contents of the final output file too:
 
-```console title="COLLECTED-output.txt"
+```console title="results/COLLECTED-output.txt"
 BONJOUR
 HELLO
 HOLà
 ```
 
-This time we have all three greetings in the final output file. Success!
+This time we have all three greetings in the final output file. Success! Remove the optional view calls to make the next outputs less verbose.
 
 !!! note
 
     If you run this several times without `-resume`, you will see that the order of the greetings changes from one run to the next.
-    This shows you that the order in which items flow through the pipeline is not guaranteed to be consistent.
+    This shows you that the order in which elements flow through process calls is not guaranteed to be consistent, unless you make use of fair threading (check the [fair](https://www.nextflow.io/docs/latest/reference/process.html#fair) directive).
```

Comment:
I don't think the mention of fair threading is helpful, it gives people too much too think about that they don't need at this stage. Let's leave that out.

Reply:
Ok, fair point 😄